network-api/networkapi/buyersguide/templates/fragments/seal-of-approval.html

```diff
@@ -1,8 +1,8 @@
 {% if product.meets_minimum_security_standards is True %}
 <img
   class="seal-of-approval"
-  src="/_images/buyers-guide/mini-badge.svg"
-  title="this product meets our minimum security standards"
-  alt="this product meets our minimum security standards"
+  src="{% if full_badge is True%}/_images/buyers-guide/badge.svg{% else %}/_images/buyers-guide/mini-badge.svg{% endif %}"
```

Comment:
Super small correction in this file (lines 4-6) for consistency. Can we make sure there's a space before an ending `%}` i.e. `{% if full_badge is True %}`.